content/applications/inventory_and_mrp/quality/quality_check_types/instructions_check.rst

```diff
@@ -0,0 +1,86 @@
+==========================
+Instructions quality check
+==========================
+
+.. |QCP| replace:: :abbr:`QCP (Quality Control Point)`
+
+In Odoo *Quality*, an *Instructions* check is one of the quality check types that can be selected
+when creating a new quality check or Quality Control Point (QCP). Instructions checks consist of a
+text entry field that allows the creator to provide instructions for how to complete the check.
+
+For a full overview of how to configure a quality check or a |QCP|, see the documentation on
+:ref:`Quality checks <quality/quality_management/quality-checks>` and :ref:`Quality Control
+Points <quality/quality_management/quality-control-points>`.
+
+Process an Instructions quality check
+=====================================
+
+There are multiple ways that Instructions quality checks can be processed. If a quality check is
+assigned to a specific manufacturing, inventory, or work order, the check can be processed on the
+order itself. Alternatively, a check can be processed from the check's page.
+
+Process from the quality check's page
+-------------------------------------
+
+To process an Instructions quality check from the check's page, begin by navigating to
+:menuselection:`Quality --> Quality Control --> Quality Checks`, and select a quality check. Follow
+the :guilabel:`Instructions` for how to complete the check.
+
+If the product passes the check, click the :guilabel:`Pass` button above the quality check form. If
+the product does not pass the check, click the :guilabel:`Fail` button, instead.
+
+Process a quality check on an order
+-----------------------------------
+
+To process an Instructions quality check on an order, select a manufacturing order or inventory
```

Comment:
> inventory order 

I don't think we have a definitive way of naming things, but I typically see the BSAs calling these "warehouse transfers" or "inventory transfers". I don't have a super strong opinion about "inventory order" but it might be confusing when linked to a sales order. 

If you do decide to change "inventory orders" to warehouse transfers, I suggest that you also change the content within the parantheses to "receipt, delivery order, return, etc."